devtools-ci-index.yaml

```diff
@@ -459,6 +459,131 @@
         - archive:
             artifacts: ./perfcake-7.4/perfcake-chart
 
+- wrapper:
+    name: generator_credentials_wrapper
+    wrappers:
+        - credentials-binding:
+            - username-password-separated:
+                credential-id: 061479ee-de4a-4fb0-b752-7b86659ba3ca
+                password: GENERATOR_DOCKER_HUB_PASSWORD
+            - file:
+                credential-id: 061479ee-de4a-4fb0-b752-7b86659ba3ca
```

Comment:
dont think you need this - this maps to the different OSD clustrs that Che is deployed to